j5/backends/console/sr/v4/power_board.py

```diff
@@ -78,8 +77,9 @@ def set_power_output_enabled(
         """Set whether a power output is enabled."""
         try:
             self._console.info(f"Setting output {identifier} to {enabled}")
+            assert identifier in self._output_states.keys()
             self._output_states[identifier] = enabled
-        except KeyError:
+        except AssertionError:
```

Comment:
Any reason for this change? I don't have anything against it really, I'm just interested to know.

Reply:
Due to the implementation of the `_output_states` on this backend, Python will not throw a `KeyError` when `self._output_states[identifier] = enabled` is called.

In order to test this properly, I added `assert identifier in self._output_states.keys()`, which is then caught when it fails.

Reply:
Ah that makes total sense, thanks!